src/raspbian/10/helix/arm32v6/Dockerfile

```diff
@@ -7,6 +7,7 @@ FROM scratch
 # But we can use a crossrootfs from our image generation scripts, and
 # extract into the image root
 
+RUN echo no-op
```

Comment:
`FROM scratch` containers (line 4) are literally empty / zero-bytes. The failure that `/bin/sh` was not found is because docker's `RUN` command is trying to invoke `/bin/sh echo no-op` here and failing to find the executable.

You can move it before `FROM scratch` (line 3).